### PR TITLE
Pac Man: Font Awesome theme

### PR DESCRIPTION
**Pull request by lindsaym-fa**
- Rounds out the Font Awesome theme with imagery, a header, and fine-tuning
- Adds a missing custom property for `--checked-icon-color` to `wa-checkbox`

### CODE REVIEW

File: src/themes/fa.css [hunk 1]
```diff
@@ -459,47 +568,73 @@ wa-button:not([variant='text']) {
...
+  & wa-rating {
+    --symbol-size: var(--wa-font-size-m);
+  }
 }
 
-wa-switch {
-  --background: var(--wa-color-neutral-spot);
-  --border-color: var(--wa-color-neutral-spot);
+wa-carousel {
+  --pagination-color-activated: var(--wa-color-brand-spot);
```
Comment (frrrances) [inline src/themes/fa.css:580]: Unrequested opinion... what if the resting color was a little lighter?
```suggestion
  --pagination-color-activated: var(--wa-color-brand-spot);
  --pagination-color-resting: var(--wa-color-gravy-80);  
```

File: src/themes/fa.css [hunk 2]
```diff
@@ -459,47 +568,73 @@ wa-button:not([variant='text']) {
...
-wa-alert {
-  --content-color: var(--wa-color-text-normal);
+wa-checkbox,
+wa-radio {
+  --checked-icon-color: var(--wa-color-surface-default);
+}
 
-  &[variant='brand'] {
-    --icon-color: var(--wa-color-brand-spot-darker);
+wa-radio-group > wa-radio-button {
```
Comment (frrrances) [inline src/themes/fa.css:589]: Love the depth on these!

File: src/themes/fa.css [hunk 3]
```diff
@@ -459,47 +568,73 @@ wa-button:not([variant='text']) {
...
-  &[variant='neutral'] {
-    --icon-color: var(--wa-color-neutral-spot-darker);
+  & wa-icon-button {
+    color: var(--wa-color-text-normal);
   }
 }
+
+wa-input > wa-icon,
+wa-select > wa-icon {
+  color: var(--wa-color-text-quiet);
```
Comment (frrrances) [inline src/themes/fa.css]: These seem a little more font-awesome-y without this override...

**Review by frrrances — COMMENTED** [review]
Looks great, @lindsaym-fa! Code is looking awesome, I left a few small optional comments for ya. 

As for the hero, I'm not 100% into it yet either. @jabronus may have some better thoughts, but here are mine in case they're helpful: 

- swap the grape for Font Awesome blue, or navy?
- bring the hero title color into more alignment with the surrounds, like tint it a little? 
- use more illustration like we do on fontawesome.com, like a variation on one of these that is more baked goods oriented?

open-source-illustration
icons-bg

Take those with a giant lump of salt (or whatever the kids are saying these days) and just use what's useful!

**Comment by jabronus** [comment]
This is looking fantastic 🤩

The think the idea of reusing some illos from the site is a great one! If you want a hand with that in any way, let me know.

The only thing that's throwing me off a tad is the depth on buttons in dark mode. It feels like a photo negative, but there may not be a way around that stylistic translation. Could also be I'm not used to seeing a dark mode FA and I just need to acclimate 😅 It doesn't have the same contrast-y punch as light mode, but I think my brain expects to see something more like this:

buttons

Also take that with a giant lump of salt!

**Comment by lindsaym-fa** [comment]
> The only thing that's throwing me off a tad is the depth on buttons in dark mode. It feels like a photo negative, but there may not be a way around that stylistic translation. Could also be I'm not used to seeing a dark mode FA and I just need to acclimate 😅 It doesn't have the same contrast-y punch as light mode, but I think my brain expects to see something more like this:
> 
> buttons


Totally, totally agree, @jabronus. I was going for "old" Web Awesome's `inverted` class, which seems to serve as a spot dark mode in FA:
Screenshot 2024-02-13 at 4 36 42 PM

I'll try out the more realistic lighting model!

**Comment by jabronus** [comment]
Ah nice, didn't know that existed! Feel free to ignore me

File: src/themes/fa.css (hunk 3 above)
Comment (lindsaym-fa) [reply src/themes/fa.css]: @frrrances, it looks like most icons in FA inputs are pretty dark...
Screenshot 2024-02-15 at 5 29 23 PM

...so I went a bit darker with my overrides.
Screenshot 2024-02-15 at 5 28 10 PM


Let me know how that sits with you!

**Review by frrrances — APPROVED** [review]
Looks amaaaazing, @lindsaym-fa! 🤩 ✨

**Pull request merged**